examples/MADDPG/README.md

```diff
@@ -0,0 +1,120 @@
+## Reproduce MADDPG with PARL
+Based on PARL, the MADDPG algorithm of deep reinforcement learning has been reproduced.
+
++ paper: 
+[ Multi-Agent Actor-Critic for Mixed Cooperative-Competitive Environments](https://arxiv.org/abs/1706.02275)
+
+### Multi-agent particle environment introduction
+A simple multi-agent particle world based on gym. Please see [here](https://github.com/openai/multiagent-particle-envs) to install and know more about the environment.
+
+### Benchmark result
```

Comment:
Could you resize the images so that it can display more images at the same line? I'm afraid it's a bit long for this readme.

Reply:
Done, 4 pictures in a row